.gitignore

```diff
@@ -5,6 +5,7 @@ node_modules
 build
 secret.json
 log.txt
+composer.lock
```

Comment:
@xaviranik `composer.lock` shouldn't be gitignored.

Reply:
fixed bhai